src/test/scala/org/broadinstitute/dsde/workbench/leonardo/TestSupport.scala

```diff
@@ -0,0 +1,11 @@
+package org.broadinstitute.dsde.workbench.leonardo
+
+import scala.concurrent.{Await, Awaitable}
+import scala.concurrent.duration.Duration
+
+/**
+  * Created by dvoet on 6/27/17.
+  */
+trait TestSupport {
```

Comment:
It looks like this trait just lets you call:
```
// waits on f1
val res1 = f1.futureValue
// waits on f2
val res2 = f2.futureValue
// waits on f3
val res3 = f3.futureValue
```
which is basically our `runAndWait` method.

http://doc.scalatest.org/3.0.0/index.html#org.scalatest.concurrent.Futures$FutureConcept